pineapple-core/src/main/java/sh/miles/pineapple/command/Command.java

```diff
@@ -129,4 +132,9 @@ public CommandLabel getCommandLabel() {
     public CommandSettings.Settings getSettings() {
         return settings;
     }
+
+    @Override
+    public boolean canUse(CommandSender sender) {
+        return sender.hasPermission(label.getPermission());
```

Comment:
if we implement this we can probably just use it above no? It'll probably look nicer